rpm/harbour-webpirate.yaml

```diff
@@ -3,9 +3,9 @@ Summary: harbour-webpirate
 Version: 1.6
 Release: 3
 # The contents of the Group field should be one of the groups listed here:
-# http://gitorious.org/meego-developer-tools/spectacle/blobs/master/data/GROUPS
+# https://gitorious.org/meego-developer-tools/spectacle/blobs/master/data/GROUPS
```

Comment:
gitorious is migrating to archive.org being read-only, but the URLs are going to stay. I wasn't able to find if meego-developer-tools is still developed and where it moved.